docs/includes/apiargs-MongoDBClient-method-construct-driverOptions.yaml

```diff
@@ -1,4 +1,58 @@
 arg_name: option
+name: autoEncryption
```

Comment:
Was this moved up to alphabetize all of the options? I noticed `allow_invalid_hostname` is still down below.

Reply:
Yes. The first commit moves the deprecated options to the bottom and sorts non-deprecated once alphabetically.